### ISSUE TITLE
Evaluate hls stream loading issues in MediaElement4

### ISSUE THREAD

**Issue by adamjarling** [avalonmediasystem/avalon#2356]
### Description
When working on the IIIF Avalon Player and consuming new manifests generated on mallorn pointing to source files such as:
```
{
   "id": "https://mallorn.dlib.indiana.edu/streams/0274365b-8eac-4a2c-8451-227cc4c61378/1ece4ade-9c05-46bf-bb05-ff7e20c4ecd3/Popeye_forPresident.mp4.m3u8?token=0add8f0ef3373927792cf4fc3689ed18bee4c553",
   "type": "Video",
   "format": "application/x-mpegURL",
   "label": "High"
}
```

, there are some hls errors firing in Chrome.  And some of the custom behavior loading a new source file, or destroying and re-initializing a Mediaelement player in the same browser window, is throwing odd errors, which I think are due to loading problems.

### Tasks
- [ ] Possible swarm topic, or schedule a meeting between front and back end devs to highlight the issue, and discuss what's going on.

**Comment by davidschober** [comment]
@adamjarling is this still valid?

**Comment by adamjarling** [comment]
@davidschober No longer valid.  Can be closed along with #2425
